pmm-tests/pmm-2-0-bats-tests/ms-specific-tests.bats

```diff
@@ -299,12 +299,12 @@ echo "$output"
         done
 }
 
-@test "run pmm-admin mysql --help check for socket" {
+@test "run pmm-admin add mysql --help check for socket" {
     run pmm-admin add mysql --help
     echo "$output"
     [ "$status" -eq 0 ]
-    echo "${output}" | grep "Usage: pmm-admin add mysql [<name> [<address>]]"
-    echo "${output}" | grep "socket=STRING"
+    [[ ${lines[0]} =~ "Usage: pmm-admin add mysql [<name> [<address>]]" ]]
+    echo "${output}" | grep -- "socket=STRING"

```

Comment:
STRING looks like a bug

Reply:
@michal-kralik @nazarovvi ^

Reply:
String is correct here. More appropriate could be a path though.

Reply:
Maybe we could create a new task for renaming hints in help to avoid merge conflicts with current PR?

Reply:
yes, let's please merge this.